clients/web/apps/docs/src/content/docs/cerebro/operations.md

```
@@ -189,38 +189,240 @@ Use this in container health checks or monitoring probes.
+### Post-Restore Verification
+
+After restoring from backup, verify service health and data integrity:
+
+1. **Check readiness endpoint:**
+   ```bash
+   curl -f http://127.0.0.1:4040/readyz
+   # Should return HTTP 200
+   ```
+
+2. **Verify memory counts:**
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)